synapseclient/table.py

```diff
@@ -771,11 +821,11 @@ def _before_synapse_store(self, syn):
         # get the default EntityView columns from Synapse and add them to the columns list
         additional_columns = []
         if self.addDefaultViewColumns:
-            additional_columns.extend(syn._get_default_entity_view_columns(self.type))
+            additional_columns.extend(syn._get_default_entity_view_columns(self.get('viewTypeMask')))
```

Comment:
use bracket notation `[]`. at this point we should be guaranteed that viewTypeMask is set. If it isnt due to an bug, we want this part of the code to error out.